admin/sales_analytics/utils.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+from admin.base.settings import ENTRY_POINTS
+from datetime import datetime, timedelta
+from framework.mongo import database as db
+
+
+# Helper functions
+def get_entry_point(system_tags, entry_points=ENTRY_POINTS):
+    """
+    Given the user system_tags, return the user entry point (osf, osf4m, prereg, institution)
+    In case of multiple entry_points existing in the system_tags, return only the first one.
+    """
+    entry_points = entry_points.keys()
+    for i in system_tags:
```

Comment:
Yes that is the assumption. Occasionally users would have more than one entry point but it is very rare (and nobody knows why). Therefore, using the first one is probably good enough.